src/Components/Aside/Aside.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import './style.css';
+import { useDataContext } from '../../Contexts/DataContext';
+import { Category } from '../Category/Category';
+import { SortByPrice } from '../Filter/SortByPrice';
+import { Slider } from '../Filter/PriceRange';
+
```

Comment:
you can make a separate file and can name it anything e.g. index.js , from that file you can export, it will be more redable

import { Slider, SortByPrice } from '../Filter/index';